cocktails/easy/wikipedia_pina_colada.md

```diff
@@ -0,0 +1,15 @@
+# Textbook (Wikipedia inspired) Piña Colada
```

Comment:
Strictly speaking this is a Painkiller, not a Piña Colada

good stuff though, this is a fab drink